yarn.lock

```diff
@@ -119,7 +119,17 @@
     file-saver "^2.0.5"
     lodash "4.17.21"
 
-"@eclipse-glsp/ide@2.1.0":
+"@eclipse-glsp/client@~2.1.0":
```

Comment:
Now all the glsp stuff seams to be there in both version 2.1.0 and 2.1.1.

Reply:
That should not cause any problems but since they are basically identical. There was just an issue with the wrong sprotty dependency, see https://github.com/eclipse-glsp/glsp-client/compare/v2.1.0...v2.1.1. But we can revert the version change as well, if you want.

Reply:
Yes, because the `2.1.0` versions have a dependency on the other `2.1.0` versions, I'll see whether we can release matching version for all bundles.